manifests/init.pp

```diff
@@ -75,6 +115,7 @@
     require => File[$nodejs_default_path]
   }
 
+  # TODO remove!
```

Comment:
I'm not a fan of this profile.d script. This template functionality fetches variables from THIS scope which makes it much harder to analyse which variables are actually needed.
Added this `TODO` comment because I'd like to find a more suitable solution in one of the next alpha releases.